wgpu-hal/src/dx12/command.rs

```diff
@@ -1401,9 +1404,12 @@ impl crate::CommandEncoder for super::CommandEncoder {
         self.prepare_dispatch([0; 3]);
         let cmd_list6: Direct3D12::ID3D12GraphicsCommandList6 =
             self.list.as_ref().unwrap().cast().unwrap();
+        let Some(command_signature) = self.shared.cmd_signatures.draw_mesh.as_ref() else {
```

Comment:
```suggestion
        let Some(ref command_signature) = self.shared.cmd_signatures.draw_mesh else {
```
Nit to keep this pattern in line with the rest of the codebase